src/opencmiss/zincwidgets/handlers/interactionmanager.py

```diff
@@ -36,23 +36,25 @@ def unregister_handler(self, handler):
     def set_fallback_handler(self, fallback_handler):
         self._fallback_handler = fallback_handler
 
+    def activate_handler(self, handler):
+        self._active_handler.leave()
+        self._active_handler = handler
+        self._active_handler.enter()
+
     def key_press_event(self, event):
-        if event.key() in self._key_code_handler_map and not event.isAutoRepeat():
+        if event.modifiers() in self._key_code_handler_map and not event.isAutoRepeat():
```

Comment:
This is not correct, this is listening for keys like `a` or `s` which the modifiers will never be.  This method doesn't listen to modfiers.